config.js

```diff
@@ -44,6 +44,10 @@ var config = {
     //
 
     testing: {
+        // Disables the End to End Encryption feature. Useful for debugging
+        // issues related to insertable streams.
+        // disableE2EE: false,
```

Comment:
is it possible to specify it through the URL params? is that something we would want? if yes the it needs to be added to the whitelist, right?

Reply:
It's already possible to do so, because `config.testing` is whitelisted.